app/models/mention.rb

```diff
@@ -1,3 +1,6 @@
+# @note When we destroy the related article, it's using dependent:
+#       :delete for the relationship.  That means no before/after
+#       destroy callbacks will be called on this object.
```

Comment:
thanks for these notes - the reason to prefer "destroy" over "delete" during development is to guard against someone adding a callback in one of the associated models - these notes provides the best effort guard to help them know not to do that (or to revisit the article deletion plan if they must).